sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.net6.0.cs

```diff
@@ -20,6 +20,19 @@ public partial class DownloadTransferValidationOptions
         public bool AutoValidateChecksum { get { throw null; } set { } }
         public Azure.Storage.StorageChecksumAlgorithm ChecksumAlgorithm { get { throw null; } set { } }
     }
+    public partial class ExpectContinueOptions
+    {
+        public ExpectContinueOptions() { }
+        public System.TimeSpan Backoff { get { throw null; } set { } }
+        public long ContentLengthThreshold { get { throw null; } set { } }
+        public Azure.Storage.ExpectContinueOptions.ApplyHeaderMode Mode { get { throw null; } set { } }
+        public enum ApplyHeaderMode
```

Comment:
Nested types are confusing for our average customer and the intellisense driven experience in C#.  The name should also be more about the user scenario enabled rather than the implementation.

Reply:
See my previous comment that this is a very specific, well-known http header that has a specific behavior. Yes it's an implementation detail but I'm really not sure it's one we want to hide. Especially since we expect most users to go along with what we set and only advanced ones may want to go in and change it.

Reply:
Currently have extracted the enum to separate file and named it `ExpectContinueOptions`. Still think we need to use a name similar to "expect continue" somewhere in this API.